interface_specifications/pe_openapi/apis/v1/a0_pe_invitation_post.md

```diff
@@ -0,0 +1,57 @@
+### Invitation POST
+
+`POST /pe/v1/invitation/`
+
+
+#### Description
+
+Any party (i.e. Verifier or Holder) can request to connect to other party. This connection invitation is sent as an out-of-band message since the DID of others may not be known at this stage.
+
+REST API acts as mediator and forwards the message to the receiver and responds back with the DID of the receiver in a DIDComm message.
+
+
+#### Request Body:
+  ```json  
+  {
+    "@type": "https://didcomm.org/out-of-band/v1/invitation",
+    "@id": "fcc9e41c-4a6d-4007-9116-ae679610d6fb",
+    "goal_code": "didcomm_connect",
+    "goal": "To connect and issue the did for communication",
+    "request~attach": [
+      {
+        "@id": "request-0",
+        "mime-type": "application/json",
+        "data": {
+          "json": {
+            "invitee": "e.g. holder 'B'",
```

Comment:
This is not necessarily known upfront. Is this a remark field, or does it expect logic to find the invitee/DID?

Reply:
It meant to identify the invitee, just like we expect someone identified by an email id. But, after discussions I understand that this may not be known upfront. I will try to change these.